### DISCUSSION TITLE
Allow multiple subscriptions for Enterprise level billing

### DISCUSSION THREAD

**Discussion by aaronwatt-marksandspencer** [community/community#14487] [Enterprise]
Currently GitHub only allows a single azure subscription to be applied for managing spending on an account. To allow splitting of costs within an enterprise it would be useful if multiple subscriptions could be added and allow specific costs e.g. Actions spending against these subscriptions. It would be helpful to have ways of allocating particular spending to these subscriptions too e.g. by Repo.

**Comment by prabhu-mns** [comment]
any feedback on this would be really great!
